typescript2/app-website/app/globals.css

```
@@ -531,8 +528,10 @@
   --input: #d8cfbd;
   --ring: #6d28d9;
 
-  background-color: var(--vsc-bg);
   color: var(--vsc-text);
+  color-scheme: light;
+
+  background-color: var(--vsc-bg);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)